lib/Monocle_Monkey.php

```diff
@@ -12,7 +12,7 @@ public function get_active($eids, $swLat, $swLng, $neLat, $neLng, $tstamp = 0, $
         $select = "pokemon_id, expire_timestamp AS disappear_time, encounter_id, lat AS latitude, lon AS longitude, gender, form";
         global $noHighLevelData;
         if (!$noHighLevelData) {
-            $select .= ", atk_iv AS individual_attack, def_iv AS individual_defense, sta_iv AS individual_stamina, move_1, move_2, cp, level";
+            $select .= ", atk_iv AS individual_attack, def_iv AS individual_defense, sta_iv AS individual_stamina, move_1, move_2, cp, level, weather_boosted_condition";
```

Comment:
Why is `weather_boosted_condition` inside `if (!$noHighLevelData)` since it's doesn't requre high level accounts?